src/app/ApplicationTemplate.Tests/Framework/DataLoader/OnDataLostDataLoaderStrategyShould.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Chinook.DataLoader;
+using FluentAssertions;
+using Xunit;
+
+namespace ApplicationTemplate.Tests.Framework
+{
+	public class OnDataLostDataLoaderStrategyShould
+	{
+		[Fact]
+		public async Task NotDisposePreviousData_When_FirstLoad()
```

Comment:
hum I figured since I'm not using the variable, I could use the discard variable. I guess the convention would make sense for the parameters. I will fix it

Reply:
fixed